polygeist/lib/Dialect/Polygeist/Transforms/LICM.cpp

```diff
@@ -1131,13 +1140,13 @@ collectHoistableOperations(LoopLikeOpInterface loop,
                }))
       continue;
 
-    const SmallVector<AccessorPairType> &AccessorPairs =
+    const SmallVector<AccessorPairType> &accessorPairs =
         candidate.getRequireNoOverlapAccessorPairs();
```

Comment:
Maybe `getRequireNoOverlapAccessorPairs` should return an `ArrayRef` too

Reply:
done.